polaris-core/src/main/java/org/apache/polaris/core/auth/PolarisAuthorizableOperation.java

```diff
@@ -121,6 +122,7 @@ public enum PolarisAuthorizableOperation {
   CREATE_CATALOG(CATALOG_CREATE),
   GET_CATALOG(CATALOG_READ_PROPERTIES),
   UPDATE_CATALOG(CATALOG_WRITE_PROPERTIES),
+  UPDATE_CATALOG_MAINTENANCE_PROPERTIES(CATALOG_WRITE_MAINTENANCE_PROPERTIES),
```

Comment:
tangential: This change LGTM, but I'm thinking about braking down the operation enum into several enums with dedicated scope (operations within a catalog, catalog management (add/remote), ermission management)... Just FYI :)